CONTRIBUTING.md

```diff
@@ -3,4 +3,4 @@
 In favor of active development we accept contributions from everyone. You can contribute by submitting a bug, creating pull requests or even by improving documentation.
 
 Below is the guide to be followed strictly before submitting your pull requests.
-http://adonisjs.com/docs/2.0/contributing
+http://adonisjs.com/docs/3.0/community
```

Comment:
@thetutlage Fixed :)

And I've changed the http://adonisjs.com/docs/installation to be without version too.
I've checked and is redirecting to http://adonisjs.com/docs/3.0/installation